gravityflow/gravityflowflowchart
# Fix Gravity Flow Flowcharts to display well with Gravity Forms 2.5

## Description
Update to Gravity Flow Flowchart to work and display correctly with Gravity Forms 2.5

## Testing instructions
1. Update the Gravity Flow Flowchart to this branch.
2. Update the Gravity Flow branch to fix-gravityforms-2.5.
3. With Gravity Forms 2.4 check if Gravity Flow Flowchart on Workflow steps displays normally as before.
4. Update to Gravity Forms 2.5 beta, and check if the Gravity Flow Flowchart displays well.

## Automated Test Enhancements
N/A
 
## Screenshots
N/A

## Documentation Changes?
N/A

## Checklist:
